nexus/db-model/src/tuf_repo.rs

```diff
@@ -303,11 +303,11 @@ impl ToSql<diesel::sql_types::Text, diesel::pg::Pg> for ArtifactHash {
     }
 }
 
-impl FromSql<diesel::sql_types::Text, diesel::pg::Pg> for ArtifactHash {
+impl FromSql<Text, diesel::pg::Pg> for ArtifactHash {
     fn from_sql(
         bytes: diesel::pg::PgValue<'_>,
     ) -> diesel::deserialize::Result<Self> {
-        let s = String::from_sql(bytes)?;
+        let s = <String as FromSql<Text, diesel::pg::Pg>>::from_sql(bytes)?;
```

Comment:
Maybe a nit, but this seems to introduce some unnecessary noise. This compiled before, so the compiler must have been able to resolve the correct method, right?

Reply:
Yeah. This was an issue in diesel 2.2, but i can stash it for now.